kind/local-env.sh

```diff
@@ -48,7 +48,7 @@ helm upgrade --install --namespace="${NAMESPACE}" netbox \
   --set externalDatabase.existingSecretName="netbox.netbox-db.credentials.postgresql.acid.zalan.do" \
   --set externalDatabase.existingSecretKey="password" \
   --set redis.auth.password="password" \
-  https://github.com/netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta.34/netbox-5.0.0-beta.34.tgz
+  https://github.com/netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta.160/netbox-5.0.0-beta.160.tgz
```

Comment:
latest just changed to 161: https://github.com/netbox-community/netbox-chart/releases/tag/netbox-5.0.0-beta.161

Reply:
LOL, on it